client/src/components/relatedProducts/Outfit.jsx

```diff
@@ -43,30 +56,59 @@ class Outfit extends React.Component {
     }));
   }
 
+  onClickLeft() {
+    if (this.state.index > 0) {
+      this.setState({
+        index: this.state.index - 1,
+      });
+    }
+  }
+
+  onClickRight() {
+    if (this.state.index < this.state.outfitProducts.length - 4) {
+      this.setState({
+        index: this.state.index + 1,
+      });
+    }
+  }
+
   // eslint-disable-next-line class-methods-use-this
   render() {
+    const { index } = this.state;
+    const endRangeLimit = this.state.outfitProducts.length - 4;
+    const productRange = this.state.outfitProducts.slice(index, index + 4);
     return (
       <div className='outfit'>
         <div>YOUR OUTFIT</div>
         <div className='card'>
           <h4>Add to Outfit</h4>
           <FontAwesomeIcon icon={['fas', 'plus']} onClick={ this.onClickPlus } />
         </div>
-        {this.state.outfitProducts.map((product) => (
+        { index ? <FontAwesomeIcon className='arrow left' data-testid='left-arrow'
+            icon={ faChevronLeft } onClick={this.onClickLeft}/> : ''
```

Comment:
Can we add the icon to our library and use the icon name here? so that we can have a central icon hub and don't need to import specific icons to each file
```suggestion
            icon='chevron-left' onClick={this.onClickLeft}/> : ''
```